jmx-metrics/docs/target-systems/activemq.md

```diff
@@ -0,0 +1,79 @@
+# ActiveMQ Metrics
+
+The JMX Metric Gatherer provides built in ActiveMQ metric gathering capabilities.
+These metrics are sourced from: https://activemq.apache.org/jmx
+
+### Metrics
+
+* Name: `activemq.consumer.count`
+* Description: The number of consumers currently reading from the broker.
+* Unit: `{consumers}`
+* Labels: `destination`
+* Instrument Type: LongUpDownCounterCallback
```

Comment:
We need to standardize these names throughout our docs, but should these be something like `ObservableLongUpDownCounter`? I'm not sure where the spec has landed.

Reply:
I'm curious about this one, can't seem to find any reference to how we should be naming it, personally, to me, it seems to make sense to match what's actually being used in the script. Is there a reason we're adding "observable"? just wasn't sure about this one. Thanks for any clarification!

Reply:
https://github.com/open-telemetry/opentelemetry-specification/blob/5d374f8045dcb24523f5f17a4b8ba94542fd0b70/specification/metrics/api.md#asynchronous-counter and the java api uses "Observable" for its measurement interfaces.